Sources/Rendering/Core/Texture/index.js

```diff
@@ -69,6 +72,8 @@ export function extend(publicAPI, model, initialValues = {}) {
     'image',
   ]);
 
+  model.viewSpecificProperties = {};
```

Comment:
It should. We may need to validate the `executive` logic to make sure it does not break on a null when trying to update its input.